Library/Homebrew/utils/buildpulse.rb

```diff
@@ -0,0 +1,43 @@
+# typed: true
+# frozen_string_literal: true
+
+module Utils
+  # Helper functions for BuildPulse integration
+  module Buildpulse
+    extend T::Sig
+
+    module_function
+
+    sig { returns(T::Boolean) }
+    def available?
```

Comment:
I'm not sure this is the best name for this method. Are there any better options?

Reply:
Maybe `enabled?` or `configured?`. Honestly, though, `available?` is fine, too.